datetime/diabetes-data.md

```diff
@@ -0,0 +1,68 @@
+
+The *storage* protocols for datetimes on diabetes devices isn't even the only source of problems in handling them. The PwD's behavior in managing their devices' display time also causes complications. The user interfaces on many diabetes devices are not designed well, so aside from inattention, errors sometimes occur because they are *easier* to make in the interface than non-errors.
+
+In general, it is good to approach diabetes device times with a very, very small set of assumptions. In other words, avoid all of [this list of assumptions](./falsehoods.md 'Falsehoods Tidepoolers (want to) believe about diabetes device times'), which have been proven false in one way or another (and often repeatedly) at Tidepool.
+
+### Handling diabetes data datetimes on ingestion
+
+If the only thing an engineer implementing the protocol to read data from a diabetes device needed to know about handling datetimes properly was "run the BtUTC algorithm," then there wouldn't be much of a need for this section. As the reader may have guessed already, BtUTC gets you most of the way there, but not all the way. There is at least one situation—plus BtUTC edge cases and bugs—that will still require the engineer working on data ingestion to manipulate diabetes device datetimes directly.
+
+#### Schedule lookup
+
+Basal schedules and other insulin pump settings are based on a schedule that is expressed in device local time. When a user changes their device's display time setting, the schedule will follow the new display time rather than shifting.
+
+The implications of this when ingesting device data are that in situations where the ingestion process needs to look up information from a currently active schedule in the currently active device settings, the ingestion engineer must translate the event's post-BtUTC `time` back to a `deviceTime` (i.e., a "display time") in order to look up which segment of the currently active schedule the event falls into. 
+
+### Handling diabetes data datetimes in the client(s)
+
+In the client, the guideline for handling diabetes data datetimes is quite simple: use the `time` field in the Tidepool data model, in combination with the user's configured timezone preferences.
+
+When using [Moment.js](https://momentjs.com/ 'Moment.js') to manipulate and/or display diabetes device datetimes using the `time` field in the Tidepool data model, in every case your usage should begin with `moment.utc(time).tz(timezone)`. For good examples of this, refer to [viz's datetime utilities](https://github.com/tidepool-org/viz/blob/master/src/utils/datetime.js '@tidepool/viz: datetime utilities').
+
+
+- - - - -
+
+[^a]: Not pathologically or maliciously badly. Just, you know, *poorly*.
+[^b]: Just kidding! We haven't actually seen this one yet at Tidepool. But it's the *only one*. Yes. *Really.*
```

Comment:
I can make this small change on this branch if @darinkrauss or whoever adds me to the docs repo as an external contributor. Or feel free to edit the branch yourselves 🙂